packages/web-components/src/components/va-pagination/va-pagination.tsx

```diff
@@ -80,21 +75,20 @@ export class VaPagination {
    * Display last page number when the page count exceeds
    * `maxPageListLength`
    */
-  @Prop() showLastPage?: boolean = false;
+  // @Prop() showLastPage?: boolean = false;
```

Comment:
Is this prop also being removed? If so, can you delete instead of commenting it out?

Also make sure to run the `yarn build` step again so that the Stencil generated file gets updated.